joomla/joomla-cms
# [4.0] Mail templates - Fix Paragraph and Remove Whitespace

Pull Request for Issue #33658

### Summary of Changes
Removed blank spaces after \n and made the paragraph break consistently with two \n


### Testing Instructions
<ol>
<li>Visit Backend</li>
<li>System -> Mail Templates</li>
<li>Check the following:
<ul>
<li> Action Logs </li>
<li> Messages: New message </li>
<li> Users: New account with admin activation </li>
<li> Users: New account with admin activation (With PW) </li>
<li> Users: Password Reset </li>
<li> Users: Request to admin to verify new account </li>
</ul>
</li>
</ol>


### Actual result BEFORE applying this Pull Request
Perfectly explained here: #33658


### Expected result AFTER applying this Pull Request
Whitespaces are removed and paragraph breaking is consistent.


### Documentation Changes Required
None
